legacy/src/main/java/org/opensearch/sql/legacy/query/DefaultQueryAction.java

```diff
@@ -110,6 +110,7 @@ public void checkAndSetScroll() {
         boolean ordered = select.isOrderdSelect();
         if (!ordered) {
           request.addSort(FieldSortBuilder.DOC_FIELD_NAME, SortOrder.ASC);
+          request.addSort("_id", SortOrder.ASC);
```

Comment:
Nit: Use constants for this string "_id"

Reply:
I was going to put it next to doc_field_name but that's not part of this repo, so I'd need another place to put it -- where's a good place?

Reply:
I think ideally we want to push for this [getting fixed upstream](https://github.com/opensearch-project/OpenSearch/blob/d6ea8eb56077de73e01d8a8a4eb11d3d4a52dea8/server/src/main/java/org/opensearch/search/sort/FieldSortBuilder.java#L102), and it will propagate back down to us. That builder class in OS seems to be where this constant is "supposed" to go. If upstream can update to a new field, that would let us remove all these refs and fix #3064 in the same breath.